conf/provisioning/dashboards/sample.yaml

```diff
@@ -9,5 +9,6 @@
 #   orgId: 1
 #   folder: ''
 #   type: file
+#   disableDeletingMissingDashbaoards: false
```

Comment:
Misspelled: disableDeletingDashboard

But I'd suggest renmaing to `deleteMissing` (default: true)

Reply:
@xlson I think the default value for bool flags should be false. since that is the default value for bools. Ex https://golang.org/pkg/crypto/tls/

How about `disableDeleteMissing` or `deleteMissingDisabled` ?

Reply:
Makes sense. I prefer `disableDeleteMissing`